mirrord/intproxy/src/proxies/simple.rs

```diff
@@ -55,8 +80,15 @@ impl BackgroundTask for SimpleProxy {
     type MessageOut = ProxyMessage;
 
     async fn run(mut self, message_bus: &mut MessageBus<Self>) -> Result<(), RequestQueueEmpty> {
+        let mut protocol_version = VERSION.clone();
```

Comment:
Should be initialized to `None`. Agent version might be lower and might not allow for batched readdir